public/sass/layout.scss

```diff
@@ -0,0 +1,53 @@
+@mixin marginSettings($top, $right, $bottom, $left) {
+  margin: $top $right $bottom $left;
+}
+
+@mixin paddingSettings($padding) {
+  padding: $padding;
+}
+
+@mixin dimensions($width, $height) {
+  height: $height;
+  width: $width;
+}
```

Comment:
might want to switch it around so that the first parameter passed is the first argument used. ie:
```@mixin dimensions($width, $height) {
width: $width;
height: $height;
}```

just a little more intuitive that way.